### DISCUSSION TITLE
.yahoofantasy file.. safe to delete?

### DISCUSSION THREAD

**Discussion by Ace7866** [mattdodge/yahoofantasy#16] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/58792096/120907030-cec36900-c623-11eb-8298-0e641e11bcde.png)

Hi again.. :)
I used the draft dump command this evening and specified the filepath with a csv (2020draft.csv) etc into the folder shown in the image above. It worked! Woo! It's like magic... In an effort to keep the folder clean, is it safe to delete the .yahoofantasy file that was created alongside the .csv? Or would this mess something up?

**Comment by mattdodge** [comment, marked as answer]
That file holds your login information and the cache of API results. So you can delete it but you may need to log in again and some of your commands may take longer to run after you do so.

Personally, I would recommend keeping it around unless it's causing you pain.

**Reply by Ace7866** [reply]
Ohhh got it. I can keep it! Thanks for the explanation.
